mtags/src/main/scala-2/scala/meta/internal/metals/ScaladocIndexer.scala

```diff
@@ -19,7 +20,7 @@ import scala.meta.pc.SymbolDocumentation
 class ScaladocIndexer(
     input: Input.VirtualFile,
     fn: SymbolDocumentation => Unit
-) extends ScalaMtags(input) {
+) extends ScalaMtags(input, Scala213) {
```

Comment:
It seems that here we can also pass the correct dialect too.
It's used only in one place and dialect might be taken from `SymbolDefinition`: https://github.com/scalameta/metals/blob/e2518701d992316d5866c5839709e02a6ea5b969/mtags/src/main/scala-2/scala/meta/internal/metals/Docstrings.scala#L82-L90 

But this is optional,  I don't mind to fix it later

Reply:
good catch, thanks! Fixed!